plugins/poll/assets/javascripts/discourse/components/modal/poll-ui-builder.hbs

```diff
@@ -38,6 +38,18 @@
           </a>
         </li>
       {{/if}}
+      {{#if this.showIrv}}
+        <li>
+          <a
+            href
+            {{on "click" (fn this.updatePollType "irv")}}
+            class="poll-type-value poll-type-value-irv
+              {{if this.isIrv 'active'}}"
+          >
+            {{i18n "poll.ui_builder.poll_type.irv"}}
+          </a>
```

Comment:
I would very much prefer a button here, links which don't navigate to a new page feel really wrong

Reply:
Done for all options: https://github.com/discourse/discourse/pull/27155/commits/35640e6374377d56403105126e74ce5bc30e1598